extensions/amp-audio/amp-audio.md

```diff
@@ -16,13 +16,60 @@ limitations under the License.
 
 ### <a name="amp-audio"></a> `amp-audio`
 
-A replacement for the HTML5 `audio` tag. Like all embedded external resources in a AMP file, the audio is lazily loaded only when the `amp-audio` element is in or near the viewport.
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>A replacement for the HTML5 `audio` tag. The `amp-audio` component is only to be used for direct HTML5 audio file embeds.</td>
```

Comment:
Here, audio and amp-audio just appear with ` around them. Did you mean to use <code>?